contracts/factory/Makefile

```diff
@@ -1,3 +1,7 @@
+ifeq (,$(BUILD_FLAGS))
+  DEPS = ../stake ../pair
```

Comment:
we no longer have pair, shouldn't this be pool?

Reply:
```suggestion
  DEPS = ../stake ../pool
```

Reply:
Good point.